pom.xml

```diff
@@ -1305,12 +1305,12 @@
                         </image>
                         <image>
                             <repository>${dockerHubPublic}/jobservice/job-service-postgres</repository>
-                            <tag>6.2.2</tag>
-                            <digest>sha256:b3d7cb9dcb8c3ad0a4e58f8e29448d61138af8cdf53711fb24811a161e574be6</digest>
+                            <tag>7.0.2</tag>
+                            <digest>sha256:676c166c8749f81c07d0e24ec6528fd8b5f3950514d9f29e6514cad64b1c0dc3</digest>
                         </image>
                         <image>
                             <repository>${dockerHubPublic}/jobservice/worker-batchsplit</repository>
-                            <tag>1.0.0</tag>
+                            <tag>1.0.0-SNAPSHOT</tag>
```

Comment:
Why is this moving to a SNAPSHOT version?

Reply:
1.0.0-SNAPSHOT also had the same digest as the 'latest' version : https://hub.docker.com/r/jobservice/worker-batchsplit/tags

Reply:
Ok.  I guess `1.0.0-SNAPSHOT` really shouldn't exist.  I'm just going to delete it.

Reply:
I've deleted it now.